src/java/main/org/apache/zookeeper/server/quorum/Leader.java

```diff
@@ -724,6 +724,8 @@ synchronized public boolean tryToCommit(Proposal p, long zxid, SocketAddress fol
        // concurrent reconfigs are allowed, this can happen.
        if (outstandingProposals.containsKey(zxid - 1)) return false;
        
+       // in order to be committed, a proposal must be accepted by a quorum
+       //
```

Comment:
@hanm I think the blank line may be fine, it provides separation from the idea on the next line so it doesn't read as a single sentence.